src/presets.ts

```diff
@@ -1016,7 +1016,7 @@ export function GetPresetList(
 		steps: [{ down: [{ actionId: ActionId.applyChannel, options: {} }], up: [] }],
 		feedbacks: [],
 	}
-	for (let index = 1; index < 8; index++) {
+	for (let index = 1; index < 9; index++) {
```

Comment:
We should make this based  on the number of outputs available in ISO, I'll make an ToDo for it